### ISSUE TITLE
getCards with single card

### ISSUE THREAD

**Issue by AntoineLemaire** [PaylineByMonext/payline-php-sdk#1]
When the wallet has severals cards, the array like this:

```
array:2 [
  "result" => array:3 [
    "code" => "02500"
    "shortMessage" => "Operation Successfull"
    "longMessage" => "Operation Successfull"
  ]
  "cardsList" => array:1 [
    "cards" => array:2 [
      0 => array:12 []
      1 => array:12 []
    ]
  ]
]
``` 

But when the wallet has only on card, we got that:

```
array:3 [
  "result" => array:3 []
  "cardsList" => array:1 [
    "cards" => array:12 [
      "walletId" => "walletID"
      "lastName" => "DUPONT"
      "firstName" => "Martin"
      "email" => "test@test.com"
      "shippingAddress" => array:7 []
      "card" => array:6 []
      "cardInd" => "1"
      "comment" => ""
      "isDisabled" => "0"
      "disableDate" => null
      "extendedCard" => array:5 []
      "default" => "Y"
    ]
  ]
  "privateDataList" => []
]
```

Why don't provide a "cards" array with one card like this?
```
array:2 [
  "result" => array:3 [
    "code" => "02500"
    "shortMessage" => "Operation Successfull"
    "longMessage" => "Operation Successfull"
  ]
  "cardsList" => array:1 [
    "cards" => array:1 [
      0 => array:12 []
    ]
  ]
]
``` 

This requires us to verify the content of the "cards" array to check if it's the array is a card, or if the array contains cards

**Comment by PaylineGitHub** [comment]
Got it ! This will be fixed in our next release.

**Comment by PaylineGitHub** [comment]
Fixed in 4.46.1. All single nodes from a list are returned with a 0-index. List nodes are : 
This applies to nodes :
	- cards (son of cardsList)
	- billingRecord (son of billingRecordList)
	- walletId (son of walletIdList)
	- transaction (son of transactionList)
	- pointOfSell (son of listPointOfSell)
	- contract (son of contracts)
	- customPaymentPageCode (son of customPaymentPageCodeList)
	- function (son of functions)
	- details (son of details)
	- privateData (son of privateDataList)
	- associatedTransactions (son of associatedTransactionsList)
	- statusHistory (son of statusHistoryList)
	- paymentAdditional (son of paymentAdditionalList)
	- CustomerTrans (son of CustomerTransHist)
	- PaymentMeansTrans (son of PaymentMeansTransHist)
	- AlertsTrans (son of AlertsTransHist)
